browser/ui/brave_ads/ad_notification_popup.cc

```diff
@@ -442,7 +461,8 @@ void AdNotificationPopup::CreateWidgetView() {
 
   widget->Init(std::move(params));
 
-  widget->SetOpacity(0.0);
+  if (!g_disable_fade_in_animation_for_testing)
+    widget->SetOpacity(0.0);
```

Comment:
Discussed via DM about always using a { } block - even for a single line

Reply:
Done